mycroft/enclosure/__init__.py

```diff
@@ -1,16 +1,2 @@
-# Copyright 2017 Mycroft AI Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from mycroft.enclosure.api import EnclosureAPI
-from mycroft.enclosure.display_manager import DisplayManager
+# backwards compat import, maintained as part of ovos_utils
```

Comment:
Should this annotate a deprecation in `0.0.3`?

Reply:
it is annotated in the main file, TODO comments are for easy grepping and not forgetting

Reply:
I thought i was answering other pr... we need to keep this around for downstreams replacing mycroft with ovos to avoid import errors, i did not add a log because we might end up keeping this around for a long time, until we stop caring about being drop in replacement....   in this last round of prs i end up adding a lot of stuff back to account for this, in general i think its fine to deprecate internal usages but i try to not break imports

Are we ok with breaking this import compat? How long do we keep this and the mk1 stuff around after phal? Do we keep dead code around just in case something is importing it until mycroft core catches up (if ever)?

Reply:
That's valid, didn't think about imports external to core, but some skills could do this. Maybe a comment to that effect makes more sense? Maybe a standard comment for all these places we can search for would help if we ever do decide to start dropping legacy support.

`# this module is no longer used in core, but unmaintained skills or plugins may import this module directly`